tasks/elasticsearch-templates.yml

```diff
@@ -29,5 +29,7 @@
   register: resultstemplate
 
 - name: Install template(s)
-  command: "curl -sL -XPUT http://localhost:{{http_port}}/_template/{{item}} -d @/etc/elasticsearch/templates/{{item}}.json"
+  command: "curl -sL -XPUT --output /dev/null --write-out %{http_code} http://localhost:{{http_port}}/_template/{{item}} -d @/etc/elasticsearch/templates/{{item}}.json"
```

Comment:
@barryib Agreed, I was just aiming for the minimal change necessary because I wasn't sure of the full context for this decision (curl vs. uri module). Anyway, I see you raised an issue to switch so I'll just close this PR.